Workiva/over_react
# Allow analyzer 1.0

## Motivation
  
Allow analyzer 1.0

## Changes
  

- Updated some ranges to get closer to resolving analyzer 1.0
- Add CI checks to validate each supported major version of analyzer

#### Release Notes
  

## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 

### QA Checklist
        - Verify all new "validate_analyzer" runs pass
        - Verify "validate_analyzer" resolves to the corresponding analyzer major version in each run (see `pub get` output)

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:


[contributing-review-types]: https://github.com/Workiva/over_react/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/over_react/blob/master/CONTRIBUTING.md#manual-testing-criteria
